client/src/pages/signupPage/SignupPage.jsx

```diff
@@ -24,8 +25,10 @@ export default function LoginPage() {
         .post(`${expressURL}/api/users/register`, data, {
           headers: { "Content-Type": "application/json" },
         })
-        .finally(() => navigate("/login"))
+
+        .finally(() => navigate("/login"));
     } catch (err) {
+      toast.error("an error occured, please try again");
```

Comment:
once you correct these, store the error message in a variable in order to eliminate possible discrepancies between them

Reply:
it's not possible as it is used once in different files